backend/endpoints/train_endpoints.py

```diff
@@ -70,8 +75,35 @@ def tabular_run():
         )
 
         try:
-            createTrainspaceData(tabular_data)
+            create_dynamo_item_from_obj("trainspace", tabular_data)
```

Comment:
what if the name of the trainspace table changes? wouldn't we need to change the `"trainspace"` parameter in instances of `create_dynamo_item_from_obj` within `train_endpoints.py`? @dwu359

Reply:
@farisdurrani what are your thoughts?

Reply:
Yes, that needs to be refactored too in some way